include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -99,46 +99,46 @@ struct SPhysicalDeviceFeatures
-    //VkBool32           shaderUniformBufferArrayNonUniformIndexing;
-    //VkBool32           shaderSampledImageArrayNonUniformIndexing;
-    //VkBool32           shaderStorageBufferArrayNonUniformIndexing;
-    //VkBool32           shaderStorageImageArrayNonUniformIndexing;
-    //VkBool32           shaderInputAttachmentArrayNonUniformIndexing;
-    //VkBool32           shaderUniformTexelBufferArrayNonUniformIndexing;
-    //VkBool32           shaderStorageTexelBufferArrayNonUniformIndexing;
-    //VkBool32           descriptorBindingUniformBufferUpdateAfterBind;
-    //VkBool32           descriptorBindingSampledImageUpdateAfterBind;
-    //VkBool32           descriptorBindingStorageImageUpdateAfterBind;
-    //VkBool32           descriptorBindingStorageBufferUpdateAfterBind;
-    //VkBool32           descriptorBindingUniformTexelBufferUpdateAfterBind;
-    //VkBool32           descriptorBindingStorageTexelBufferUpdateAfterBind;
-    //VkBool32           descriptorBindingUpdateUnusedWhilePending;
-    //VkBool32           descriptorBindingPartiallyBound;
-    //VkBool32           descriptorBindingVariableDescriptorCount;
+    bool descriptorIndexing = false;
+    bool shaderInputAttachmentArrayDynamicIndexing = false;
+    bool shaderUniformTexelBufferArrayDynamicIndexing = false;
+    bool shaderStorageTexelBufferArrayDynamicIndexing = false;
+    bool shaderUniformBufferArrayNonUniformIndexing = false;
+    bool shaderSampledImageArrayNonUniformIndexing = false;
+    bool shaderStorageBufferArrayNonUniformIndexing = false;
+    bool shaderStorageImageArrayNonUniformIndexing = false;
+    bool shaderInputAttachmentArrayNonUniformIndexing = false;
+    bool shaderUniformTexelBufferArrayNonUniformIndexing = false;
+    bool shaderStorageTexelBufferArrayNonUniformIndexing = false;
+    bool descriptorBindingUniformBufferUpdateAfterBind = false;
+    bool descriptorBindingSampledImageUpdateAfterBind = false;
+    bool descriptorBindingStorageImageUpdateAfterBind = false;
+    bool descriptorBindingStorageBufferUpdateAfterBind = false;
+    bool descriptorBindingUniformTexelBufferUpdateAfterBind = false;
+    bool descriptorBindingStorageTexelBufferUpdateAfterBind = false;
+    bool descriptorBindingUpdateUnusedWhilePending = false;
+    bool descriptorBindingPartiallyBound = false;
+    bool descriptorBindingVariableDescriptorCount = false;
     //VkBool32           runtimeDescriptorArray; // [FUTURE TODO]
     
     bool                 samplerFilterMinmax = false;   // ALIAS: VK_EXT_sampler_filter_minmax
     
-    //VkBool32           scalarBlockLayout;     // or VK_EXT_scalar_block_layout // [FUTURE TODO]
+    //VkBool32           scalarBlockLayout;     // or VK_EXT_scalar_block_layout
     
-    //VkBool32           imagelessFramebuffer;  // or VK_KHR_imageless_framebuffer 
+    //VkBool32           imagelessFramebuffer;  // or VK_KHR_imageless_framebuffer // [FUTURE TODO]
```

Comment:
there's actually an issue about it now #378 

The plan is that we'll "rewrite the API" and basically require that feature at some point.